packages/wdio-appium-service/src/launcher.js

```diff
@@ -48,8 +49,12 @@ export default class AppiumLauncher {
     }
 
     _startAppium(command, args, waitStartTime, callback) {
-        log.debug(`Will spawn Appium process: ${command} ${args.join(' ')}`)
-        let process = spawn(command, args, { stdio: ['ignore', 'pipe', 'pipe'] })
+        const cmd = isWindows ? 'cmd' : command
```

Comment:
Is this the best place to do this? It seems like you'll be overwriting any command passed through the [onPrepare method](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-appium-service/src/launcher.js#L24-L30).

Reply:
I think so, Windows commands need to be started different in comparison to Mac/Linux

Reply:
@wswebcreation I think what @baruchvlz was saying is whether we could move this to `onPrepare`

Reply:
Moved it, also gave me the option to mock the `process.platform`